modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/VerticalCardTag.java

```diff
@@ -368,6 +383,7 @@ protected String getHydratedModuleName() {
 
 	@Override
 	protected Map<String, Object> prepareProps(Map<String, Object> props) {
+		props.put("ariaLabel", getAriaLabel());
```

Comment:
if `getAriaLabel()` returns `null`, this will send a null value to the serializer. It's probably safer to avoid sending this prop if  its value is null

Reply:
Regarding this case, I'm not sure. Most of the rest of props have the same behavior (they are optional), and we are not checking them before sending. Anyway, the react component doesn't add the attribute if the value is null, so I think it's safe to leave it that way here, what do you think?